api/src/main/java/javax/config/spi/ConfigBuilder.java

```diff
@@ -77,6 +77,8 @@
     /**
      * Add the specified {@link ConfigSource}.
      *
+     * Attention: If you register the same {@code ConfigSource} instance within multiple {@link Config} then non-portable behaviour results.
```

Comment:
results -> occurs

Reply:
No, this is the language also throughout the CDI spec.